test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x/ActionResult/ActionResultController.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
Not a big deal but the [ActionResult](https://github.com/RamjotSingh/WebApi/tree/master/test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x/ActionResult) directory would typically go here [\WebApi\test\E2ETest\Microsoft.Test.E2E.AspNet.OData](https://github.com/RamjotSingh/WebApi/tree/master/test/E2ETest/Microsoft.Test.E2E.AspNet.OData). That way the source files would be attached as links to any of the test projects in either [Build.AspNet](https://github.com/RamjotSingh/WebApi/tree/master/test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet), [Build.AspNetCore](https://github.com/RamjotSingh/WebApi/tree/master/test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore), or [Build.AspNetCore3x](https://github.com/RamjotSingh/WebApi/tree/master/test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x) as the case may be

Reply:
This actually is intentional because ActionResult is only supported NetCore2.2 and above but since both NetCore22 and NetCore30 are out of support, that makes the first version where ActionResult is supported as NetCore31 hence I have put it under the appropriate folder. The test is also for the same reason not added to rest of the 2 projects.